src/swap/utils/processSwapTransaction.test.ts

```diff
@@ -120,10 +134,9 @@ describe('processSwapTransaction', () => {
       statusName: 'transactionPending',
     });
     expect(updateLifecycleStatus).toHaveBeenNthCalledWith(4, {
-      statusName: 'transactionApproved',
+      statusName: 'success',
```

Comment:
shouldn't we still be emitting transactionApproved here before success?

Reply:
good catch, updated in this PR